lib/quda_matrix.h

```diff
@@ -958,45 +958,59 @@ namespace quda {
     return true;
   }
 
+  template<class Cmplx>
+  __device__ __host__
+    double ErrorSU3(const Matrix<Cmplx,3>& matrix)
+    {
+      const Matrix<Cmplx,3> identity_comp = conj(matrix)*matrix;
+      double error;
```

Comment:
Was just about to merge this, then I noticed a bug here: `error` hasn't been initialized so could lead to nonsense results....